README.md

```diff
@@ -1,6 +1,29 @@
 # kube-scheduler-wasm-extension
 
- All the things to make the scheduler extendable with wasm.
+[WebAssembly](https://webassembly.org/) is a way to safely run code compiled in other
+languages. Runtimes execute WebAssembly Modules (Wasm), which are most often
+binaries with a `.wasm` extension.
+This project allows you to extend the kube-scheduler with custom scheduler plugin compiled to a Wasm
+binary. It works by embedding a WebAssembly runtime, [wazero](https://wazero.io), into the
+scheduler, and loading custom scheduler plugin via configuration.
+
+This project contains everything needed to extend the scheduler:
+- Documentation describing what type of actions are possible, e.g. `Filter`.
+- Language SDKs used to build scheduler plugins, compiled to wasm.
+- The scheduler plugin which loads and runs wasm plugins
+
+## Motivation
```

Comment:
here's a rephrase of your motivation in perhaps more natural English:

Today, you can extend the kube-scheduler, but it is non-trivial. To customize the scheduler means writing Go, and a non-trivial build process. Once you've built your scheduler, you have deployment and configuration work to have your cluster use it. This isn't one time work, as it needs to be redone on every upgrade of your cluster.

This project lowers that tension by making the default scheduler capable of loading custom plugins, compiled to WebAssembly (wasm). This removes the deployment burdens above. It also allows plugins to be written in languages besides Go.

Reply:
Great, I fixed based on your suggestion.